feature-libs/cart/base/components/cart-shared/cart-item-list/cart-item-list.component.ts

```diff
@@ -56,6 +56,8 @@ export class CartItemListComponent implements OnInit, OnDestroy {
 
   @Input() readonly: boolean = false;
 
+  @Input() disableLink: boolean = false;
```

Comment:
I had to add a new toggle to inactive product link. On Inspect Cart page, user should not be able to leave the page including navigation to pdp.
Even though navigation guard would prevent it.
I belive it is not a breaking change as this new toggle is optional.